src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/ValueContentAnalysis/ValueContentAbstractValue.cs

```diff
@@ -154,7 +154,32 @@ protected override void ComputeHashCodeParts(ref RoslynHashCode hashCode)
         protected override bool ComputeEqualsByHashCodeParts(CacheBasedEquatable<ValueContentAbstractValue> obj)
```

Comment:
❓ Can you review the complexity of changing the `ComputeEqualByHashCodeParts` methods to consider both `HasValue` and `Value` when the input is a `Nullable<T>` instance?

Reply:
@sharwell This is not viable as I found it's easy to still get collisions even when null isn't involved. See https://github.com/dotnet/roslyn-analyzers/pull/6876#discussion_r1299392581

Reply:
@sharwell This is not viable as I found it's easy to still get collisions even when null isn't involved. See https://github.com/dotnet/roslyn-analyzers/pull/6876#discussion_r1299392581